### PR TITLE
Introduce Core monthly generator

### PR DESCRIPTION
**Pull request by kpodemski**
Core monthly generator via: `php bin/console github:monthly:report`

### CODE REVIEW

**Review by matks — COMMENTED** [review]
Looks good so far 👍

File: src/App/Command/GitHubMonthlyReportCommand.php [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    {
+        $reportFilename = 'core-monthly-' . $this->dateStart . '-' . $this->dateEnd . '.md';
+        $reportPath = $input->getOption('outputDir') . DIRECTORY_SEPARATOR . $reportFilename;
+
+        $this->searchForIssuesData();
+        $this->searchForPullRequests();
+        $this->searchForReleases();
+
+        $output->writeln('Writing monthly report...');
+        $template = file_get_contents('var/data/templates/core-monthly.md');
```
[the remark this answers is not in the document]
Comment (kpodemski) [reply src/App/Command/GitHubMonthlyReportCommand.php:150]: One day @matks, one day there will be Twig! ;) for now, it is as simple as possible

**Comment by kpodemski** [comment]
@matks I think we can go for merge, and yes that latest monthly base on this code :)

for the `core-weekly-generator`, read-only + disable gh action that generates it weekly

**Comment by matks** [comment]
Let's go 🚀

**Comment by matks** [comment]
> for the `core-weekly-generator`, read-only + disable gh action that generates it weekly

Repo disabled and PR to disable GA is https://github.com/PrestaShop/prestashop.github.io/pull/1162/

**Pull request merged**